docs/subnets/troubleshoot-deployments.md

```diff
@@ -0,0 +1,95 @@
+[~]$ avalanche subnet deploy mySubnet
+✔ Local Network
+Deploying [mySubnet] to Local Network
+Backend controller started, pid: 26388, output at: /Users/user/.avalanche-cli/runs/server_20221231_111605/avalanche-cli-backend
+VMs ready.
+Starting network...
+..................................................................................
+..................................................................................
+......Error: failed to query network health: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+```
+
+Avalanche-CLI only supports running one local Avalanche network at a time. If other instances of
+AvalancheGo are running concurrently, your Avalanche-CLI network fails to start.
+
+To test for this error, start by shutting down any Avalanche nodes started by Avalanche-CLI.
+
+```shell
+avalanche network clean --hard
+```
+
+Next, look for any lingering AvalancheGo processes with:
+
+```shell
+ps aux | grep avalanchego
+```
+
+If any processes are running, you need to stop them before you can launch your VM with Avalanche-CLI.
+
+:::warning
+
+If you're running a validator node on the same box you're using Avalanche-CLI, **don't** end any
+of these lingering AvalancheGo processes. This may shut down your validator and could affect
+your validation uptime.
+
+:::
+
+## Incompatible RPC Version for Custom VM
+
+If you're locally deploying a custom VM, you may run into this error message.
+
+```text
+[~]$ avalanche subnet deploy mySubnet
+✔ Local Network
+Deploying [mySubnet] to Local Network
+Backend controller started, pid: 26388, output at: /Users/user/.avalanche-cli/runs/server_20221231_111605/avalanche-cli-backend
+VMs ready.
+Starting network...
+.........
+Blockchain has been deployed. Wait until network acknowledges...
+..................................................................................
+..................................................................................
+......Error: failed to query network health: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+```
+
+This error has many possible causes, however you should be aware of the possibility of an RPC
+protocol version  mismatch.
+
+AvalancheGo communicates with custom VMs over RPC using [gRPC](https://grpc.io/). gRPC defines a
+protocol specification shared by both AvalancheGo and the VM. Both components **must** be running
+the same RPC version for VM deployment to work. You can view [AvalancheGo's RPC compatibility broken
+down by release
+version](https://github.com/ava-labs/avalanchego/blob/master/version/compatibility.json).
+
+Your custom VM's RPC version is set by the version of AvalancheGo that you import. By default,
+Avalanche-CLI creates local Avalalanche networks that run the latest AvalancheGo release.
+
+Here's an example with real numbers from the AvalancheGo compatibility page. If the latest
+AvalancheGo release is version v1.9.4, then Avalanche-CLI deploys a network with RPC version 21. For
```

Comment:
probably confusing 21 with 20 in `with RPC version 21`

Reply:
fixed